projects/exui-common-lib/src/lib/gov-ui/components/hmcts-pagination/hmcts-pagination.component.html

```diff
@@ -0,0 +1,44 @@
+<pagination-template #p="paginationApi" [id]="id" [maxSize]="maxSize" (pageChange)="pageChange.emit($event)"
+  (pageBoundsCorrection)="pageBoundsCorrection.emit($event)">
+
+  <nav class="hmcts-pagination" id="pagination-label">
+    <p class="govuk-visually-hidden" aria-labelledby="pagination-label">Pagination navigation</p>
+    <ul class="hmcts-pagination__list" *ngIf="!(p.pages.length <= 1)">
+
+      <li class="hmcts-pagination__item  hmcts-pagination__item--prev" [class.disabled]="p.isFirstPage()">
+        <a [routerLink]="" class="hmcts-pagination__link" *ngIf="1 < p.getCurrent()" (keyup.enter)="p.previous()"
+          (click)="p.previous()">Previous <span class="govuk-visually-hidden">set of pages</span></a>
+        <span *ngIf="p.isFirstPage()">
+          Previous <span class="govuk-visually-hidden">set of pages</span>
+        </span>
+      </li>
+
+      <ng-container *ngIf="showPageNumbers">
+        <li class="hmcts-pagination__item" [class.hmcts-pagination__item--active]="p.getCurrent() === page.value"
+          [class.hmcts-pagination__item--dots]="page.label === '...'" *ngFor="let page of p.pages">
+          <a class="hmcts-pagination__link" [routerLink]="" (keyup.enter)="p.setCurrent(page.value)"
+            (click)="p.setCurrent(page.value)" *ngIf="p.getCurrent() !== page.value">
```

Comment:
@paulhowes-HMCTS - as above for the previous `*ngIf` comment.

Reply:
My reply is the same as above

Reply:
Resolving as per above responses.